file_store/src/lora_valid_poc.rs

```diff
@@ -40,31 +41,30 @@ impl MsgDecode for LoraValidPoc {
 impl TryFrom<LoraValidPocV1> for LoraValidPoc {
     type Error = Error;
     fn try_from(v: LoraValidPocV1) -> Result<Self> {
-        let beacon_report: LoraValidBeaconReport = v.beacon_report.unwrap().try_into()?;
-        let mut witnesses: Vec<LoraValidWitnessReport> = Vec::new();
-        for witness_proto in v.witness_reports {
-            let witness_report: LoraValidWitnessReport = witness_proto.try_into()?;
-            witnesses.push(witness_report)
-        }
+        let witnesses = v
+            .witness_reports
+            .into_iter()
+            .map(TryFrom::try_from)
```

Comment:
Stop using TryFrom::try_from.. you can be specific in what you're calling try_from on

Reply:
Fixed c086696